tests/Constraints/OfPropertiesTest.php

```diff
@@ -72,22 +72,22 @@ public function getInvalidTests()
                 null,
                 array(
                     array(
-                        "property"   => "prop2",
-                        "pointer"    => "/prop2",
-                        "message"    => "Array value found, but a string is required",
-                        "constraint" => "type",
+                        'property' => 'prop2',
```

Comment:
Yes there is, but since the majority of the codebase did not align the double arrow in array notations, I opted to make this consistent everywhere. If however it is desired that these be aligned, then the rule can simply be changed to that and the rest of the code will be adjusted accordingly.